charactersheet/charactersheet/models/common/armor.js

```diff
@@ -25,7 +25,12 @@ function Armor() {
     self.armorStealthOptions = ko.observableArray(Fixtures.armor.armorStealthOptions);
 
     self.acLabel = ko.pureComputed(function() {
-        return 'AC ' + self.armorClass();
+        if (self.armorClass().replace(/^\s+|\s+$/g, '')) {
```

Comment:
Correct, but this behavior is consistent. We should discuss if this is something we want to address, since it's a bit of work.